openssl/src/ssl/test.rs

```diff
@@ -1038,6 +1041,13 @@ fn tmp_dh_callback() {
 
     let stream = TcpStream::connect(("127.0.0.1", port)).unwrap();
     let mut ctx = SslContext::builder(SslMethod::tls()).unwrap();
+    // TLS 1.3 has no DH suites, and openssl isn't happy if the max version has no suites :(
```

Comment:
Is this a temporary problem?

Reply:
Dunno

Reply:
I'm actually not sure why the ECDHE tests passes - the OpenSSL docs claim that the TLS 1.3 suites aren't in the groups that we're activating.

Reply:
Oh right, we only run that on 1.0.1 and 1.0.2.